lib/RNSingleSelect.tsx

```diff
@@ -90,13 +91,18 @@ const RNSingleSelect = (props: IProps) => {
     spinnerType = "ThreeBounce",
     spinnerSize = 30,
     spinnerColor,
+    value = null
     searchEnabled = true
   } = props;
 
+  const getInitialSelectedItemValue = () => {
+    return value ? value : null;
+  };
+
```

Comment:
Actually, we do not need this function, it is default value is already set to `null` which's why this is redundant.

Reply:
Yeah, you're totally right